addons/stock/models/stock_picking.py

```diff
@@ -264,6 +264,7 @@ class Picking(models.Model):
 
     move_line_ids = fields.One2many('stock.move.line', 'picking_id', 'Operations')
     move_line_ids_without_package = fields.One2many('stock.move.line', 'picking_id', 'Operations without package', domain=['|',('package_level_id', '=', False), ('picking_type_entire_packs', '=', False)])
+    move_line_nosuggest_ids = fields.One2many('stock.move.line', 'picking_id', domain=[('product_qty', '=', 0.0)])
```

Comment:
i don't get it, this field was brought back by the first revert of this pr isn't it?

Reply:
The revert gets back it on `stock.move`, and I added it also on `stock.picking` to don't display existing moves on *detailed operations* in the picking form view.